returnn/datasets/sprint.py

```diff
@@ -64,9 +64,12 @@ def __init__(self, target_maps=None, str_add_final_zero=False, input_stddev=1.,
     :param None|dict[str] bpe: if given, will be opts for :class:`BytePairEncoding`
     :param None|dict[str] orth_vocab: if given, orth_vocab is applied to orth and orth_classes is an available target`
     :param bool suppress_load_seqs_print: less verbose
+    :param int|None reduce_target_factor: downsample factor to allow less targets than features
```

Comment:
```suggestion
    :param int reduce_target_factor: downsample factor to allow less targets than features
```
I don't think None is allowed.